### PR TITLE
Replace RC with Deployments in quick-start.md and deploying-applications.md

### PR DESCRIPTION
**Pull request by janetkuo**
Updated docs in http://janetkuo.github.io/docs/user-guide/#thorough-walkthrough item 1~3 by replacing RC with Deployments. 

Also combine `kubectl run` and `kubectl expose` as `kubectl run --expose`.

@kubernetes/docs @pwittrock

### CODE REVIEW

File: docs/user-guide/quick-start.md [hunk 1]
```diff
@@ -60,12 +52,11 @@ automatically.
 To kill the application and delete its containers and public IP address, do:
 
 ```shell
-$ kubectl delete rc my-nginx
-replicationcontrollers/my-nginx
-$ kubectl delete svc my-nginx
-services/my-nginx
+$ kubectl delete deployment/my-nginx service/my-nginx
```
Comment (bgrant0607) [inline docs/user-guide/quick-start.md]: We should be able to do kubectl delete deployment,service my-nginx

**Comment by johndmulhausen** [comment]
Just great work, @janetkuo; so far you're definitely winning the award for "fastest adopter of new docs conventions." :) I had @bgrant0607 take a quick peek for tech review's sake. Everything LGTM. Merging.

File: docs/user-guide/quick-start.md (hunk 1 above)
Comment (johndmulhausen) [inline docs/user-guide/quick-start.md]: Woops, was about to merge before I saw this inline comment. Will wait until this command is formatted to be comma separated.

Comment (janetkuo) [inline docs/user-guide/quick-start.md]: Done

**Comment by janetkuo** [comment]
@johndmulhausen the comment is addressed and PTAL :smile:

**Comment by johndmulhausen** [comment]
Thanks, Janet! Looks great. :)

**Pull request merged**